staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3453,10 +3454,16 @@ func describeCertificateSigningRequest(csr metav1.ObjectMeta, signerName string,
 			return
 		}
 		w.Write(LEVEL_0, prefix+name+":\t")
-		w.Write(LEVEL_0, strings.Join(values, "\n"+prefix+"\t"))
+		w.Write(LEVEL_0, "%q", strings.Join(values, "\n"+prefix+"\t"))
 		w.Write(LEVEL_0, "\n")
 	}
 
+	CommonName := cr.Subject.CommonName
```

Comment:
If this is a normal variable with limited scope, shouldn't it be named something like - `commonName` instead of `CommonName`? I have a very limited experience with Go, but have normally seen people using`commonName` pattern, so I was curious.

(Don't mind me, I was looking through good-first-issues and came across the issue this PR is fixing, so I thought I'd see how it was fixed and this thing caught my eye)

Reply:
It has be yes. the variable is not exported outside. I dont mind thanks :)